packages/lesswrong/lib/collections/pagecache/collection.ts

```diff
@@ -0,0 +1,19 @@
+import schema from './schema';
+import { createCollection } from '../../vulcan-lib';
+import { addUniversalFields } from '../../collectionUtils'
+import { ensureIndex } from '../../collectionIndexUtils';
+
+export const PageCache: PageCacheCollection = createCollection({
+  collectionName: 'PageCache',
+  typeName: 'PageCacheEntry',
+  collectionType: 'pg',
+  schema,
+  logChanges: true,
```

Comment:
I'm sort of worried about bloating `LWEvents` if we're gonna log changes to cached page renders (which are presumably very big, relatively speaking)

Reply:
Good point, I'll change this